### DISCUSSION TITLE
Documentation - shareConfig not published at https://rxjs.dev/

### DISCUSSION THREAD

**Discussion by miiihi** [ReactiveX/rxjs#6985] [Report issues other than bug]
The v7 share operator is replacing all multicast operators, but it's very important shareConfig parameter / interface documentation is not published. Although jsdoc of the interface exists in the source coude.

And further, the defaults of shareConfig are not documented even in the jsdoc, had to look them up source code.

I guess it both issues are very important and should be fixed asap!

**Comment by jakovljevic-mladen** [comment]
This is fixed in #6943 and it's now published to [the official docs](https://rxjs.dev/api/index/interface/ShareConfig). Thanks for reporting.

**Reply by miiihi** [reply]
Thanks for fixing, but the second part, the defaults for shareConfig are still not documented. 
I guess this is even more important, as they are quite unintuitive.

**Reply by jakovljevic-mladen** [reply]
I agree. The [share](https://rxjs.dev/api/operators/share) docs states that the default `ShareConfig` object is `{}` (if nothing is provided when using `share()` like this 👈). It _might_ be good enough, but maybe it isn't: there is no info about what happens internally on [this line](https://github.com/ReactiveX/rxjs/blob/718be5bb8db5bb07ec9e4da8b5741e579bba6e62/src/internal/operators/share.ts#L144).

I'll try to add it to the docs.

**Reply by miiihi** [reply]
Exactly, the mentioned line is the default for share and not {} as documented.
Especially unintuitive with all boolean reset fields, which are true by default and not false as one would expect.

**Reply by jakovljevic-mladen** [reply]
@miiihi, I created [a PR](https://github.com/ReactiveX/rxjs/pull/7050) that should solve the issue described here. Thanks a lot for reporting.
